src/packages/board/board.py

```diff
@@ -97,6 +111,19 @@ def __init__(self, api_handler=None):
         """
         Internal counter for tracking used indexes
         """
+        self._offline = offline
+
+        super(RatBoard, self).__init__()
+
+    async def on_online(self):
+        logger.info("Board moving to online mode...")
```

Comment:
Please note tense is important.  'Board now online' or 'Board moved to online mode'